docs/monitors/jenkins.md

```diff
@@ -0,0 +1,160 @@
+ - ***`jenkins.node.executor.in-use`*** (*gauge*)<br>    Total number of executors being used in an instance
+ - ***`jenkins.node.health-check.score`*** (*gauge*)<br>    Mean health score of an instance
+ - ***`jenkins.node.queue.size`*** (*gauge*)<br>    Total number pending jobs in queue
+ - ***`jenkins.node.vm.memory.heap.usage`*** (*gauge*)<br>    Percent utilization of the heap memory
+ - ***`jenkins.node.vm.memory.non-heap.used`*** (*gauge*)<br>    Total amount of non-heap memory used
+ - ***`jenkins.node.vm.memory.total.used`*** (*gauge*)<br>    Total Memory used by instance
+
+#### Group node/status
+All of the following metrics are part of the `node/status` metric group. All of
+the non-default metrics below can be turned on by adding `node/status` to the
+monitor config option `extraGroups`:
+ - ***`jenkins.node.online.status`*** (*gauge*)<br>    Boolean value of instance is reachable or not
+
+#### Group slave/status
+All of the following metrics are part of the `slave/status` metric group. All of
+the non-default metrics below can be turned on by adding `slave/status` to the
+monitor config option `extraGroups`:
+ - ***`jenkins.node.slave.online.status`*** (*gauge*)<br>    Boolean value for slave is reachable or not
+
+### Non-default metrics (version 4.7.0+)
+
+**The following information applies to the agent version 4.7.0+ that has
+`enableBuiltInFiltering: true` set on the top level of the agent config.**
+
+To emit metrics that are not _default_, you can add those metrics in the
+generic monitor-level `extraMetrics` config option.  Metrics that are derived
+from specific configuration options that do not appear in the above list of
+metrics do not need to be added to `extraMetrics`.
+
+To see a list of metrics that will be emitted you can run `agent-status
+monitors` after configuring this monitor in a running agent instance.
+
+### Legacy non-default metrics (version < 4.7.0)
+
+**The following information only applies to agent version older than 4.7.0. If
+you have a newer agent and have set `enableBuiltInFiltering: true` at the top
+level of your agent config, see the section above. See upgrade instructions in
+[Old-style whitelist filtering](../legacy-filtering.md#old-style-whitelist-filtering).**
+
+If you have a reference to the `whitelist.json` in your agent's top-level
+`metricsToExclude` config option, and you want to emit metrics that are not in
+that whitelist, then you need to add an item to the top-level
+`metricsToInclude` config option to override that whitelist (see [Inclusion
+filtering](../legacy-filtering.md#inclusion-filtering).  Or you can just
+copy the whitelist.json, modify it, and reference that in `metricsToExclude`.
+
+## Dimensions
+
+The following dimensions may occur on metrics emitted by this monitor.  Some
+dimensions may be specific to certain metrics.
+
+| Name | Description |
+| ---  | ---         |
+| `Job` | Name of the job. |
```

Comment:
Lowercase dimension names.

Reply:
Copied this from collectd metadata , this whole list is inaccurate and needs to be update.
I noticed other monitors don't populate this list, is there a reason for that?

Reply:
There is a JIRA issue created to go back and fill in the other monitors as well.

Reply:
change is made